templates/internet-of-things/gateways.html

```diff
@@ -1,387 +1,673 @@
-      <p class="p-heading--3">EdgeX Foundry</p>
-      <p>EdgeX Foundry is a vendor-neutral open source project hosted by The Linux Foundation building a common open framework for IoT edge computing. </p>
-      <a href="https://snapcraft.io/edgexfoundry">EdgeX in the Snap store</a>
-    </div>
-    <div class="col-6 u-hide--medium u-hide--small u-align--center">
-      {{
-        image(
-          url="https://dashboard.snapcraft.io/site_media/appmedia/2018/12/icon_Hx6IyH0.png",
-          alt="",
-          height="120",
-          width="120",
-          hi_def=True,
-          loading="lazy"
-        ) | safe
-      }}
-    </div>
-  </div>
-</section>
+  {% include "internet-of-things/shared/_try-ubuntu-core.html" %}
 
-<section class="p-strip">
-  <div class="row u-equal-height">
-    <div class="col-4 u-hide--medium u-hide--small u-align--left">
-      {{
-        image(
-        url="https://assets.ubuntu.com/v1/c4b290c8-Contact+us.svg",
-        alt="",
-        height="280",
-        width="280",
-        hi_def=True,
-        ) | safe
-      }}
-    </div>
-    <div class="col-8 u-vertically-center">
-      <div>
-        <h2>Get in touch!<br />We are innovators, just like you.</h2>
-        <p>Our team will help you build secure and flexible gateways for the industrial edge.</p>
-        <p><a class="p-button--positive js-invoke-modal" href="/internet-of-things/contact-us?product=gateways">Get in touch</a></p>
+  <section class="p-section">
+    <div class="row--50-50">
+      <hr class="p-rule" />
+      <div class="col">
+        <h2>Sign up for our IoT newsletter</h2>
       </div>
+      <div class="col">{% include "shared/contextual_footers/_iot_newsletter_signup.html" %}</div>
     </div>
-  </div>
-</section>
-
-{% with current="gateways", strip_class="p-strip--light" %}
-  {% include "internet-of-things/shared/_links.html" %}
-{% endwith %}
-
-{% with first_item="_iot_developers", second_item="_iot_newsletter_signup", third_item="_iot_further_reading" %}{% include "shared/contextual_footers/_contextual_footer.html" %}{% endwith %}
-
-<!-- Set default Marketo information for contact form below -->
-<div class="u-hide" id="contact-form-container" data-form-location="/shared/forms/interactive/internet-of-things" data-form-id="1663" data-lp-id="3143" data-return-url="https://ubuntu.com/internet-of-things/thank-you?product=gateways" data-lp-url="https://pages.ubuntu.com/things-contact-us.html">
-</div>
+  </section>
+
+  {% with title_link="/blog/internet-of-things", groupId="1666" %}
+    {% include "internet-of-things/shared/_latest-news-iot.html" %}
+  {% endwith %}
+
+  <!-- Set default Marketo information for contact form below-->
+  <div class="u-hide"
+       id="contact-form-container"
+       data-form-location="/shared/forms/interactive/internet-of-things"
+       data-form-id="2639"
+       data-lp-id="5811"
+       data-return-url="https://ubuntu.com/internet-of-things/thank-you?product=appstore"
```

Comment:
Shouldn't it be:
`data-form-id="1663"`
`data-lp-id="3143"`
`data-return-url="https://ubuntu.com/internet-of-things/thank-you?product=gateways"`

If yes, please restore this branch, commit and merge back to feature-iot-rebrand. Thanks!